### PR TITLE
Removing duplicated code in clsRlink

### LINKED ISSUE #6622: Remove duplicated code in clsRlink
In PR #6590 , @shadrackkibet suggested some code quality improvements for clsRLink.vb (see screenshot below).

image

### PR DESCRIPTION
**Pull request by Ivanluv**
fixes #6622
@africanmathsinitiative/developers  this is ready for review

### CODE REVIEW

**Comment by shadrackkibet** [comment]
@N-thony please could you review?

**Review by N-thony — COMMENTED** [review]
Looking good from my end. Over to you @shadrackkibet. Thanks.

File: instat/clsRLink.vb [hunk 1]
```diff
@@ -2199,12 +2095,16 @@ Public Class RLink
     ''' <returns>   The red flag column names. </returns>
     '''--------------------------------------------------------------------------------------------
     Public Function GetRedFlagColumnNames(strDataName As String) As String()
+        Return GetColumnNames(strDataName, "$get_red_flag_column_names")
+    End Function
+
+    Private Function GetColumnNames(strDataFrameName As String, strRCommand As String) As String()
```
Comment (shadrackkibet) [inline instat/clsRLink.vb:2101]: Could you add the XML header to this new function?

**Review by shadrackkibet — CHANGES_REQUESTED** [review]

File: instat/clsRLink.vb (hunk 1 above)
Comment (Ivanluv) [reply instat/clsRLink.vb:2117]: Done

**Review by shadrackkibet — APPROVED** [review]
@lloyddewit could you also review and merge.

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@shadrackkibet I just made some small updates to the function headers. Please could you approve again and merge? thanks

**Review by shadrackkibet — APPROVED** [review]

**Pull request merged**